### PR TITLE
Feature#29 generate qr codes

### LINKED ISSUE #29: Generate QR Codes
# REQUIREMENTS

Generate a public view that shows the qr of the different places so that any user can register their visit from the mobile application.

# DESIGN

Does not apply

# Acceptance Criteria

- [x] Dynamic search 
- [x] Qr generator

### PR DESCRIPTION
**Pull request by mariormt17**
# Description
- In this PR we implemented the rqrcode gem to dynamically generate QR codes to register the visit or entry to a place.
- In the QR codes the id of the place showed was saved.
 
 # Testing
- Just enter a specific place
  
  ## Screenshots 
imagen

### CODE REVIEW

File: alerta-covid/app/models/place.rb [hunk 1]
```diff
@@ -1,4 +1,21 @@
...
+      color: 'black',
+      file: nil,
+      fill: 'white',
+      module_px_size: 6,
+      resize_exactly_to: false,
+      resize_gte_to: false,
+      size: 120
+    )
+    IO.binwrite("app/assets/images/qrcode.png", png.to_s)
+  end
```
Comment (ArielMD) [inline alerta-covid/app/models/place.rb]: You can try to save the image with a unique path, so you don't overwrite it, for example ```"app/assets/images/qrcode_#{place_id}.png"```

File: alerta-covid/app/models/place.rb [hunk 2]
```diff
@@ -1,4 +1,21 @@
 class Place < ApplicationRecord
   belongs_to :area
   validates :name, :area, presence: true
+
```
Comment (ArielMD) [inline alerta-covid/app/models/place.rb:4]: you can add after_save: generate_qr_code, so for each places created, you generate its qr code

**Review by ArielMD — DISMISSED** [review]
Good job, I liked the QR code implementation

File: alerta-covid/app/models/place.rb (hunk 1 above)
Comment (mariormt17) [reply alerta-covid/app/models/place.rb]: Good idea, I will implement it😁

File: alerta-covid/app/models/place.rb (hunk 2 above)
Comment (mariormt17) [reply alerta-covid/app/models/place.rb:4]: In the controller I call this method each time show the place

File: alerta-covid/app/controllers/places_controller.rb [hunk 3]
```diff
@@ -8,6 +8,7 @@ def index
 
   # GET /places/1 or /places/1.json
   def show
+    Place.generate_qr_code(params[:id])
```
Comment (MiguelOcegueraM) [inline alerta-covid/app/controllers/places_controller.rb:11]: nice method!

**Review by MiguelOcegueraM — APPROVED** [review]
Nice code! I think you should delete the gems in your Gemfile that you aren't using

File: alerta-covid/app/controllers/places_controller.rb (hunk 3 above)
Comment (mariormt17) [reply alerta-covid/app/controllers/places_controller.rb:11]: Thanks!

**Review by ArielMD — APPROVED** [review]
Nice

**Review by ramasoftdev — APPROVED** [review]
Excelent Work.

**Pull request merged**